engine/src/Interpreter/interpreter_ops.cuh

```diff
@@ -450,6 +450,7 @@ private:
 			}	else if(right_position == SCALAR_INDEX) {
 				if (is_string_type(right_type_id)) {
 					right_str_view = static_cast<cudf::string_scalar_device_view*>(scalars_right[op_index])->value();
+					right_valid = true;
```

Comment:
I think this line is not necessary, `right_valid` is set to true in line 457
```suggestion
```

Reply:
Right, removed deprecated change, thanks!